### ISSUE TITLE
The end of the plugins?

### ISSUE THREAD

**Issue by Twonox** [Ribesg/NPlugins#246]
Why there is no update?

**Comment by Ribesg** [comment]
Bukkit is dead.

**Comment by Twonox** [comment]
There is Spigot, you can't use?

**Comment by Ribesg** [comment]
Spigot is not Bukkit. The Spigot implementation is also a fork of CraftBukkit, which is illegal to own, use and rely on. I'm not dealing with all of that. I no longer have time for all that bullshit.

**Comment by stuntguy3000** [comment]
Citation needed on the illegal part.

A majority of (Craft)Bukkit plugins work using the Spigot fork.

**Comment by Ribesg** [comment]
Damn, not this again. I'm tired of explaining it again and again, you either accept it or refuse to accept it, the argument never works.

TL;DR Bukkit came to an end, Spigot decided to steal the work of the Bukkit team, Wolverness used the fact that the CraftBukkit license was invalid (aka every single CraftBukkit line is owned by its author, because that's how the "no license" license works) to forbid ownership, use, and everything about CraftBukkit, which he was the main contributor to.

Counter-arguments: "but Spigot replaced all his code in their fork!"
* No, re-committing the exact same code or even derived code under your own name doesn't give you ownership of that code.
* Even if it did, no, removing the code that you are forbidden to use doesn't change anything. You were not allowed to have CraftBukkit, you were not allowed to modify it, you were not allowed to publish your changes. You were not allowed to do anything, and no future event other than Wolverness giving you the right to do what you want would change this.

Damn, I did it again.
To all the people still saying Spigot is valid to use, **fuck you**. It's not. **It's a fucking fact**. It's **not** open to discussion.

Note: These plugins *do* work on Spigot as far as they didn't break Bukkit. I'm just not going to work on it anymore, as I have problems with what the Spigot team did and I respect both Wolverness and the Bukkit team's decision.

**Comment by stuntguy3000** [comment]
@Ribesg Mate, you seriously need to chill out about this. It is actually open to discussion.
